SU2_DOT/src/SU2_DOT.cpp

```diff
@@ -267,9 +267,16 @@ int main(int argc, char *argv[]) {
       if (rank == MASTER_NODE) cout << "Reading surface sensitivities at each node from file." << endl;
       geometry_container[iZone][INST_0]->SetBoundSensitivity(config_container[iZone]);
     } else {
-      if (rank == MASTER_NODE) cout << "Reading volume sensitivities at each node from file." << endl;
+
+      if (rank == MASTER_NODE)
+        cout << "Reading volume sensitivities at each node from file." << endl;
       grid_movement[iZone] = new CVolumetricMovement(geometry_container[iZone][INST_0], config_container[iZone]);
-      geometry_container[iZone][INST_0]->SetSensitivity(config_container[iZone]);
+
+      /*--- Read in sensitivities from file. ---*/
+      if (config_container[ZONE_0]->GetDesign_Variable(0) == EXTERNAL_SENSITIVITY)
+        geometry_container[iZone][INST_0]->ReadExternalSensitivity(config_container[iZone]);
```

Comment:
Could you give some explanation where the main difference lies between _ReadExternalSensitivity_ and _SetSensitivity_? From looking through the code, I see that there are some changes concerning the filename, but I guess I'm missing something important here.
Where should I focus on?

Reply:
Alright, so it's primarily about the file format, thanks for clarifying.

Still I find the term "external sensitivities" to describe the sensitivity input (in the config file as well as in the code) a bit misleading. It is somehow suggesting that the sensitivities in SU2_CFD's output files were no external sensitivities to SU2_DOT in that regard (which they are).

Something like "UNORDERED_SENS_FILE" would be more appropriate, I think. It's not as elegant, though. Maybe someone comes up with a good name?
I could also think about one myself if we would generally agree on changing it (I'm just proposing it :)).

Reply:
Yep, basically just another format that SU2_DOT can handle for the same process we already perform with our native ASCII and binary files.

I am open name changes, of course, if there are any other ideas!

Reply:
What do you think about "PROJECT_SENSITIVITIES" or "SURFACE_SENSITIVITY"?

Maybe @talbring has an idea, too?
As it's just up to a name, I'd like to give it a small push so that we can merge it in :+1:

Reply:
@oleburghardt : you are right, this is a subtle but important change. I actually like your first suggestion for 'UNORDERED_SENS_FILE' as opposed to the SU2 native format. I am going to slightly change the interface here too in another commit soon so that it is more clear.

Reply:
I have adjusted the interface now and think this is even more useful. References to 'external sensitivity' have been removed in favor of reusing the same 'SURFACE_FILE' DV option that already exists for SU2_DEF. 

The envisioned usage is that one can use the DV type 'SURFACE_FILE' to create their own workflow with a custom parameterization outside of the current options in SU2. In this case, the user would export the projected surface sensitivities at each surface node from SU2_DOT, use their own parameterization/optimizer to decide the displacements for each surface node, and then provide those surface node displacements to SU2_DEF for mesh deformation before running the next primal/adjoint.

I have also separated the idea of the unordered sensitivity file. Now, for any parameterization in SU2_DOT, users can decide whether to use the NATIVE_BINARY format for the volume sensitivity reader, or to use the new UNORDERED_ASCII. There is a new option for this, but the default is NATIVE_BINARY.

Thanks for the feedback and let me know what you think, Ole.

Lastly, I have updated the Xcode projects for the other modules (thanks again for the reminder, Jessie!)